src/PrestaShopBundle/Routing/LegacyControllerConstants.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Routing;
+
+class LegacyControllerConstants
+{
+    public const CLASS_ATTRIBUTE = '_controller_class';
```

Comment:
```suggestion
    public const CLASS_ATTRIBUTE = '_legacy_controller_class';
```
I think we didn't understand each other with my last comment, using `legacy` as prefix everywhere seems appropriate, to avoid confusing ith Symfony attributes